Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -337,6 +347,11 @@ static size_t computeRangeSize(uint64_t size, size_t offset)
     m_buffer.label = label;
 }
 
+uint64_t Buffer::size() const
+{
+    return m_emptyBuffer.size() ?: m_size;
```

Comment:
is `?:` standard C++? I thought it was a GNU extension but maybe thing have changed.

Reply:
I don't think it is, but we are in Objective-C++ so I think this is ok as it is widely used in Objective-C/C++

I can also avoid using it too if you prefer

Reply:
Oh duh, my bad. Failed to notice this was a mm file :)